tests/python/openassetio/test/manager/resources/plugins/StubManager.py

```diff
@@ -50,9 +50,11 @@ def info(self):
         return self.__info
 
     def settings(self, hostSession):
+        # pylint: disable=unused-argument
         return self.__settings
 
     def initialize(self, managerSettings, hostSession):
+        # pylint: disable=unused-argument
```

Comment:
[fix] Is this needed? We look to use both args now?

Reply:
:confused: good spot, wonder why I put that there.

Reply:
Removed in 4104ea14